README.rst

```diff
@@ -0,0 +1,181 @@
+Zipline
+=======
+
+|Gitter|
+|version status|
+|downloads|
+|build status|
+|Coverage Status|
+|Code quality|
+
+Zipline is a Pythonic algorithmic trading library. The system is
+fundamentally event-driven and a close approximation of how live-trading
+systems operate.
+
+Zipline is currently used in production as the backtesting engine
+powering `Quantopian Inc. <https://www.quantopian.com>`__ -- a free,
+community-centered platform that allows development and real-time
+backtesting of trading algorithms in the web browser.
+
+`Join our
+community! <https://groups.google.com/forum/#!forum/zipline>`__
+
+Want to contribute? See our `open
+requests <https://github.com/quantopian/zipline/wiki/Contribution-Requests>`__
+and our `general
+guidelines <https://github.com/quantopian/zipline#contributions>`__
+below.
+
+Features
+========
+
+-  Ease of use: Zipline tries to get out of your way so that you can
+   focus on algorithm development. See below for a code example.
+
+-  Zipline comes "batteries included" as many common statistics like
+   moving average and linear regression can be readily accessed from
+   within a user-written algorithm.
+
+-  Input of historical data and output of performance statistics is
+   based on Pandas DataFrames to integrate nicely into the existing
+   Python eco-system.
+
+-  Statistic and machine learning libraries like matplotlib, scipy,
+   statsmodels, and sklearn support development, analysis and
+   visualization of state-of-the-art trading systems.
+
+Installation
+============
+
+The easiest way to install Zipline is via ``conda`` which comes as part
+of `Anaconda <http://continuum.io/downloads>`__ or can be installed via
+``pip install conda``.
+
+Once set up, you can install Zipline from our Quantopian channel:
+
+::
+
+    conda install -c Quantopian zipline
+
+Currently supported platforms include:
+
+-  Windows 32-bit (can be 64-bit Windows but has to be 32-bit Anaconda)
+
+-  OSX 64-bit
+
+-  Linux 64-bit
+
+PIP
+---
+
+Alternatively you can install Zipline via the more traditional ``pip``
+command. Since zipline is pure-python code it should be very easy to
+install and set up:
+
+::
+
+    pip install numpy   # Pre-install numpy to handle dependency chain quirk
+    pip install zipline
+
+If there are problems installing the dependencies or zipline we
+recommend installing these packages via some other means. For Windows,
+the `Enthought Python
```

Comment:
I don't think anyone we know of uses Enthought successfully with Zipline.  We should probably point people on Windows to Anaconda.